doc/modules/neighbors.rst

```diff
@@ -252,6 +252,62 @@ the lower half of those faces.
     multi-output regression using nearest neighbors.
 
 
+Nearest Centroid Classifier
```

Comment:
Fixed. 

Yes, as per https://github.com/scikit-learn/scikit-learn/issues/4521#issuecomment-89848613 its in the right place above the details on the trees and neighbors implementations.